src/composables/useFetchLatestVersion.ts

```diff
@@ -0,0 +1,68 @@
+import { ref } from "vue";
+import semver from "semver";
+import { UpdateInfo as UpdateInfoObject } from "@/type/preload";
+
+// 最新版があるか調べる
+export const useFetchLatestVersion = () => {
+  const isCheckingFinished = ref<boolean>(false);
+  const currentVersion = ref("");
+  const latestVersion = ref("");
```

Comment:
これ元々のコードがそうだった形なのですが、より分かりやすくしておくのも良いかもですね。
`latestVersion`（最新バージョン）はどっちかというと`newerLatestVersion`（より新しい最新バージョン）のがあってるかも？

まあそこ変更するのは大変かもなので、こんな感じでドキュメントに追加しておくのはどうでしょう！
```suggestion
// 最新版があるか調べる
// 現バージョンより新しい最新版があれば`latestVersion`に代入される
export const useFetchLatestVersion = () => {
  const isCheckingFinished = ref<boolean>(false);
  const currentVersion = ref("");
  const latestVersion = ref("");
```